nautobot_golden_config/tests/test_api.py

```diff
@@ -103,6 +104,14 @@ def setUp(self):
         self.add_permissions("nautobot_golden_config.add_goldenconfigsetting")
         self.add_permissions("nautobot_golden_config.change_goldenconfigsetting")
         self.base_view = reverse("plugins-api:nautobot_golden_config-api:goldenconfigsetting-list")
+        self.content_type = ContentType.objects.get(app_label="dcim", model="device")
```

Comment:
Can we add a test or two for still using scope?

Reply:
For testing if scope is supplied now that we have a setter?